apps/machine/zynqmp_r5/platform_info.c

```diff
@@ -229,6 +229,13 @@ int platform_poll(void *priv)
 
 	prproc = rproc->priv;
 	while(1) {
+#ifdef RPMSG_NO_IPI
+		char * done_polling = (char*)POLL_SHM_LOCATION;
+		if (*done_polling){
```

Comment:
direct access without memory region management as done on Linux part?

Reply:
similar as previous comments, rework to use metal_io_region

Reply:
not fixed

Reply:
this is outdated commit now. I have updated using io_region instead in new commits